references/depth/stereo/presets.py

```diff
@@ -0,0 +1,144 @@
+from typing import Optional, Tuple, Union
+
+import torch
+import transforms as T
+
+
+class StereoMatchingEvalPreset(torch.nn.Module):
+    def __init__(
+        self,
+        mean: float = 0.5,
+        std: float = 0.5,
+        resize_size: Optional[Tuple[int, int]] = None,
+        max_disparity: Optional[float] = 512,
```

Comment:
I think for inference, it might be better to have `None` as default value for max_disparity (so we dont assume on the image's zoom level). What do you think?